discovery-provider/alembic/versions/42d5afb85d42_create_audio_transactions_history.py

```diff
@@ -45,7 +45,7 @@ def upgrade():
             ),
             sa.Column("change", sa.Numeric(), nullable=False),
             sa.Column("balance", sa.Numeric(), nullable=False),
-            sa.Column("tx_metadata", sa.String(), nullable=False),
+            sa.Column("tx_metadata", sa.String(), nullable=True),
```

Comment:
double checking - this migration hasn't run yet on prod?

also, has it run on staging?

Reply:
it hasn't run anywhere yet.